src/components/modal/index.tsx

```diff
@@ -88,7 +102,7 @@ class Modal extends Component<ModalProps> {
     const Container: any = blurView ? blurView : defaultContainer;
 
     return (
-      <RNModal visible={Boolean(visible)} {...others}>
+      <RNModal visible={Boolean(visible)} supportedOrientations={SUPPORTED_ORIENTATIONS} {...others}>
```

Comment:
It seems like it's a breaking change, passing all orientation options here overrides the Navigation layout orientation config.